app/src/main/java/com/work/restaurant/data/source/remote/FitnessCenterDataImpl.kt

```diff
@@ -1,56 +1,48 @@
 package com.work.restaurant.data.source.remote
 
 import com.work.restaurant.network.api.FitnessCenterApi
-import com.work.restaurant.network.model.FitnessCenterItemModel
+import com.work.restaurant.network.model.FitnessCenterItemResponse
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
-import retrofit2.Retrofit
-import retrofit2.converter.gson.GsonConverterFactory
 
-class FitnessCenterDataImpl : FitnessCenterData {
+class FitnessCenterDataImpl private constructor(private val fitnessCenterApi: FitnessCenterApi) :
+    FitnessCenterData {
     override fun getFitnessCenterData(
         callback: FitnessCenterDataCallback
     ) {
 
 
-        val retrofit = Retrofit.Builder()
-            .baseUrl(URL)
-            .addConverterFactory(GsonConverterFactory.create())
-            .build()
+        fitnessCenterApi.fitnessCenterAllItem()
+            .enqueue(object : Callback<List<FitnessCenterItemResponse>> {
 
+                override fun onFailure(
+                    call: Call<List<FitnessCenterItemResponse>>?,
+                    t: Throwable?
+                ) {
+                    callback.onFailure("$t")
```

Comment:
```suggestion
                    callback.onFailure("$t.message")
```
실패한 내용을 볼 수 있는 변수입니다.!

Reply:
b2cff38 반영하였습니다.